packages/instant/src/components/buy_button.tsx

```diff
@@ -43,7 +43,7 @@ export class BuyButton extends React.Component<BuyButtonProps> {
                 onClick={this._handleClick}
                 isDisabled={shouldDisableButton}
                 fontColor={ColorOption.white}
-                fontSize="20px"
+                fontSize="16px"
```

Comment:
Since we went from 20 -> 16 in a few places, I'm wondering if we should introduce this number as a constant so we only have to change it one place in the future?  Not a huge deal tho